src/types.ts

```diff
@@ -84,6 +84,11 @@ export interface MonacoEditorProps extends MonacoEditorBaseProps {
    * An event emitted when the content of the current model has changed.
    */
   onChange?: ChangeHandler;
+
+  /**
+   * A flag which tells if editor content updated via value prop should get selected (default true)
```

Comment:
I feel there must be more to it. If not, we should name the property differently.

Reply:
only pushEditOperations preserves the undo stack: https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.itextmodel.html#pusheditoperations

I don't think I like this change then. Maybe it's better to just expect people to call setValue manually (which is pretty easy since you get the editor handle when you add the editor, no?)

Reply:
Hmm, makes sense. This is a very specific requirement which can be done using the editor instance (via ref).
Closing this PR, thanks!

Reply:
Thanks for the pull request and for thinking through the issue with me.